gradio/helpers.py

```diff
@@ -33,7 +33,10 @@
 if TYPE_CHECKING:  # Only import for type checking (to avoid circular imports).
     from gradio.components import Component
 
-CACHED_FOLDER = "gradio_cached_examples"
+CACHED_FOLDER = str(
+    Path(os.environ.get("GRADIO_EXAMPLES_CACHE") or "gradio_cached_examples").resolve()
+)
```

Comment:
I think we can just do:
```suggestion
CACHED_FOLDER = os.environ.get("GRADIO_EXAMPLES_CACHE", "gradio_cached_examples")
```
since we resolve it to an absolute path later when we use this.